webapp/graphite/finders/standard.py

```diff
@@ -80,8 +82,15 @@ def _find_paths(self, current_dir, patterns):
     else:
       entries = [ pattern ]
 
-    subdirs = [entry for entry in entries if isdir(join(current_dir, entry))]
-    matching_subdirs = match_entries(subdirs, pattern)
+    if using_globstar:
+        matching_subdirs = map(operator.itemgetter(0), os.walk(current_dir))
```

Comment:
According to the [docs](https://docs.python.org/3/library/os.html#os.walk), os.walk returns a list of tuples like `(dirpath, dirnames, filenames)`, so I figured that first item we are getting is always a dir, and doesn't need an `isdir` check.